### PR TITLE
Engfis 1432

### PR DESCRIPTION
**Pull request by delsborg**
# What's new?
SOLR is now pulled and build from scratch from the VIVO SOLR repo. 
SOLR is upgraded to version 8 from version 7
Moved config info from docker compose files to the .env files which is more appropriate
Updated README

# How should this be tested?
Follow build and deploy steps in README

# This is deployed to all 3 pre-production instances on prometheus-dev

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,7 @@
 FROM tomcat:9-jdk11-openjdk
 
-ARG SOLR_URL=http://localhost:8983/solr/vivocore
+#ARG SOLR_URL=http://localhost:8983/solr/vivocore
```
Comment (dottermann) [inline Dockerfile]: Is there a reason this comment needs to persist?

**Review by dottermann — COMMENTED** [review]
Overall looks pretty good but I couldn't tell you too much about what the changes are doing. A walkthrough would likely be the most beneficial for me.

Has this been deployed to non-prod?

File: Dockerfile (hunk 1 above)
Comment (delsborg) [reply Dockerfile]: Nope, I will remove

**Review by dottermann — APPROVED** [review]
LGTM. Approved

**Pull request merged**